packages/hebao_v1/contracts/modules/security/GuardianUtils.sol

```diff
@@ -35,105 +35,71 @@ library GuardianUtils
         }
 
         // Calculate total group sizes
-        Data.Guardian[] memory allGuardians = securityStore.guardians(wallet);
-        uint[MAX_NUM_GROUPS] memory total = countGuardians(allGuardians);
+        Data.Guardian[] memory allGuardians = securityStore.guardians(wallet, false);
+        require(allGuardians.length > 0, "NO_GUARDIANS");
```

Comment:
There has no limit when remove guardian and all guardians can be removed, if then, the exceed quota transfer would be fail here.